app/models/pglogical_subscription.rb

```diff
@@ -102,6 +102,9 @@ def validate(new_connection_params = {})
 
   def backlog
     connection.xlog_location_diff(remote_node_lsn, remote_replication_lsn)
+  rescue => e
```

Comment:
Is there something more specific we can rescue here? Maybe a `PG` error of some sort?

Reply:
yeaa, I should have done it